tests/simulator_testfixture.h

```diff
@@ -721,6 +721,384 @@ if __name__ == '__main__':
+  gates.push_back(GateRY<fp_type>::Create(24, 2, 6.9));
+  gates.push_back(GateRZ<fp_type>::Create(24, 3, 7.0));
+  gates.push_back(GateRX<fp_type>::Create(24, 4, 7.1));
+  gates.push_back(GateRY<fp_type>::Create(24, 5, 7.2));
+  gates.push_back(GateHd<fp_type>::Create(25, 5).ControlledBy({4}));
+  gates.push_back(GateRX<fp_type>::Create(26, 0, 7.3));
+  gates.push_back(GateRY<fp_type>::Create(26, 1, 7.4));
+  gates.push_back(GateRZ<fp_type>::Create(26, 2, 7.5));
+  gates.push_back(GateRX<fp_type>::Create(26, 3, 7.6));
+  gates.push_back(GateRY<fp_type>::Create(26, 4, 7.7));
+  gates.push_back(GateRZ<fp_type>::Create(26, 5, 7.8));
+
+  StateSpace state_space(1);
+  Simulator simulator(1);
+
+  auto state = state_space.Create(num_qubits);
+  std::vector<std::vector<fp_type>> final_amplitudes = {
+    {0.04056215, 0.11448385},
+    {0.04013729, -0.061976265},
+    {0.05715254, 0.06587616},
+    {-0.0999089, -0.0551068},
+    {-0.020135913, -0.0017108098},
+    {-0.056598634, -0.011147065},
+    {-0.05639626, 0.09074731},
+    {-0.057448477, 0.040516872},
+    {0.0344304, 0.016834},
+    {-0.0556134, -0.006876275},
+    {-0.036210306, -0.045713138},
+    {0.106739536, 0.04557059},
+    {0.0042791665, 0.071074575},
+    {-0.025317883, 0.06527158},
+    {0.003052316, -0.002724175},
+    {-0.027759908, 0.082198195},
+    {-0.10696569, 0.009430081},
+    {-0.03781139, 0.11874371},
+    {-0.020180658, -0.07570377},
+    {0.05576851, -0.022236263},
+    {-0.06552034, 0.058305625},
+    {-0.0484216, -0.1268896},
+    {-0.088334806, -0.2118823},
+    {-0.058212772, -0.10756658},
+    {0.06811757, -0.10867228},
+    {-0.006912032, -0.056490533},
+    {0.14454205, -0.08358974},
+    {0.09103435, 0.15097837},
+    {-0.023433153, -0.11143835},
+    {0.019963266, -0.0008750437},
+    {0.25689512, -0.13761702},
+    {0.060466085, -0.083674595},
+    {-0.10356863, -0.031856094},
+    {0.05267005, -0.040480673},
+    {0.0017506611, -0.057084523},
+    {-0.049090747, 0.0076575093},
+    {-0.05804465, 0.048070334},
+    {-0.037869103, 0.007335903},
+    {-0.13274089, -0.1556583},
+    {-0.013423506, -0.10376227},
+    {0.063333265, -0.20126863},
+    {-0.1259143, 0.07443194},
+    {0.13821091, 0.045418564},
+    {0.034076303, 0.054569334},
+    {-0.09922538, -0.09469399},
+    {0.09066829, -0.064125836},
+    {0.235489, -0.19617496},
+    {0.15996316, -0.036261443},
+    {-0.02887804, -0.047851864},
+    {0.046452887, -0.05820565},
+    {0.015137469, 0.07583993},
+    {-0.09476741, -0.054346137},
+    {0.015158612, 0.08472719},
+    {-0.03694186, 0.0070148334},
+    {-0.025821798, 0.08404015},
+    {0.061565418, -0.012411967},
+    {-0.078881726, 0.12779479},
+    {-0.05464944, 0.056015424},
+    {-0.16184065, -0.009010859},
+    {0.12749553, -0.12438276},
+    {0.019615382, 0.092316},
+    {-0.04924332, 0.044155773},
+    {-0.24133444, -0.033628717},
+    {-0.18774915, 0.12311842},
+  };
+
+  for (unsigned i = 0; i < size; i++) {
```

Comment:
Replace `i++` with `++i` for consistency with the rest of the code?

Reply:
Done.